web/client/epics/login.js

```diff
@@ -101,6 +103,42 @@ export const redirectOnLogout = action$ =>
         .filter(({ redirectUrl }) => redirectUrl)
         .switchMap(({ redirectUrl }) => Rx.Observable.of(push(redirectUrl)));
 
+/**
+ * Verifies the session from the cookie.
+ * This is present if login has been done using OpenID.
+ * @memberof epics.login
+ * @return {external:Observable} emitting login success or logout events if the cookie is valid.
+ */
+export const verifyOpenIdSessionCookie = (action$, {getState = () => {}}) => {
```

Comment:
we stopped using Epic in the name of epics?

Reply:
Actually it is not a strict coding convention